static/js/zamboni/devhub.js

```diff
@@ -787,7 +787,7 @@ function initPayments(delegate) {
         "img/zamboni/contributions/after.png",
         "img/zamboni/contributions/roadblock.png",
     ],
-        media_url = $("body").attr("data-media-url"),
+        static_url = $("body").attr("data-static-url"),
         to = false,
         img = $("<img id='contribution-preview' alt='' />");
         moz = $("input[value='moz']");
```

Comment:
That's a var, because the previous line is a , - but I'll note line 792 ends in a ; , meaning 793 is a global. I can change it to be a var per line.